pysb/tests/test_integrate.py

```diff
@@ -109,14 +111,29 @@ def test_integrate_with_expression():
     Rule('R2', None >> s20(), ks0)
     Rule('R3', s16() + s20() >> s16() + s1(), keff)
 
-    time = np.linspace(0, 40)
+    tspan = np.linspace(0, 40)
 
-    solver = Solver(model, time)
+    solver = Solver(model, tspan)
     solver.run()
 
-    assert solver.yexpr_view.shape == (len(time),
+    assert solver.yexpr_view.shape == (len(tspan),
                                        len(model.expressions_dynamic()))
-    assert solver.yobs_view.shape == (len(time), len(model.observables))
+    assert solver.yobs_view.shape == (len(tspan), len(model.observables))
+
+@with_model
+def test_integrate_with_time():
```

Comment:
The `Solver` class that this file tests should have been deprecated in favor of the newer and better-designed Simulator classes, and in fact Solver just delegates to `ScipyOdeSimulator` now anyway. Could you move this test to `test_simulator_scipy.py`, and replace the Solver call with the following (including a check on the simulation result):
```python
    sim = ScipyOdeSimulator(model, tspan=tspan)
    simres = sim.run()
    assert np.isclose(simres.species[-1, 0], <expected final concentration of A to 6 figures>)
```